sign/mayo/internal/common/nist/drbg.go

```diff
@@ -0,0 +1,71 @@
+// Package nist implements helpers to generate NIST's Known Answer Tests (KATs).
```

Comment:
`github.com/cloudflare/circl/internal/nist` does not implement `io.Reader`, which was somehow used during the debug process when implementing MAYO. Can change back to this one.

Reply:
Please ignore the previous comment. I think it would be better if `github.com/cloudflare/circl/internal/nist` implements `io.Reader` because MAYO's sign takes a random source which is used during KAT tests.